tests/elements-regression/src/pages/editor-page.js

```diff
@@ -90,7 +90,7 @@ module.exports = class EditorPage extends BasePage {
 		return this.page.frame( { name: 'elementor-preview-iframe' } );
 	}
 
-	async getPreviewElement( id ) {
```

Comment:
I think it should be `async await`. And in `getPreviewFrame()` as well.

Reply:
I don't thinks so - `getPreviewFrame` doesn't return or use a Promise and neither does `locator`